.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "ipfs"
```

Comment:
should eslint-config-ipfs be included in devDeps?

Reply:
Not necessary, it's included by aegir.